source/features/dim-bots.tsx

```diff
@@ -4,30 +4,37 @@ import * as pageDetect from 'github-url-detection';
 
 import features from '.';
 
-const botSelector = [
+const commitSelector = [
 	/* Commits */
 	'.commit-author[href$="%5Bbot%5D"]',
 	'.commit-author[href$="renovate-bot"]',
-	'.commit-author[href$="scala-steward"]',
+	'.commit-author[href$="scala-steward"]'
+].join();
 
+const prSelector = [
 	/* Issues/PRs */
 	'.opened-by [href*="author%3Aapp%2F"]',
 	'.labels [href$="label%3Abot"]'
 ].join();
 
 function init(): void {
-	for (const bot of select.all(botSelector)) {
+	for (const bot of select.all(commitSelector)) {
 		// Exclude co-authored commits
-		if (select.all('a', bot.parentElement!).every(link => link.matches(botSelector))) {
+		if (select.all('a', bot.parentElement!).every(link => link.matches(commitSelector))) {
 			bot.closest('.commit, .Box-row')!.classList.add('rgh-dim-bot');
 		}
 	}
+
+	for (const bot of select.all(prSelector)) {
+		bot.closest('.commit, .Box-row')!.classList.add('rgh-dim-bot');
+	}
 }
 
 void features.add(__filebasename, {
 	include: [
 		pageDetect.isCommitList,
-		pageDetect.isConversationList
+		pageDetect.isConversationList,
+		pageDetect.isPRList
```

Comment:
PRList is also already a conversation list if I remember correctly

Reply:
Doesn't seem so, see https://github.com/fregante/github-url-detection/blob/master/index.ts#L205